source/php/Module/views/partials/collection.blade.php

```diff
@@ -0,0 +1,15 @@
+@collection([
+    'classList' => ['o-grid', 'o-grid--horizontal'],
+    'attributeList' => [
+        'js-like-container' => '',
+        'js-display-as' => $displayAs,
+        'js-post-types' => $postTypes,
+        'js-columns' => $postColumns,
+        'js-like-emblem-url' => $emblem
+    ]
+])
+    @for ($i = 0; $i < 4; $i++)
+        <div class="liked-posts__preloader u-preloader u-preloader__opacity--7 u-rounded {{ $postColumns }}"
+            style="height:170px;width:100%;"></div>
```

Comment:
I agree, however im not sure how it can be done better. We can add a class in the modules css, but this seems unnecessary. Also there are no utilities for setting a specific height